randovania/data/json_data/prime1/Phendrana Drifts.txt

```diff
@@ -212,15 +212,24 @@ Asset id: 3006924320
   > Door to Canyon Entryway
       Trivial
   > Middle-Left Rooftop
-      Any of the following:
-          All of the following:
-              Morph Ball Bomb and Morph Ball and L-Jump (Beginner) and Slope Jump (Beginner) and Standable Terrain (Beginner) and Bomb Jump (Beginner)
-              Any of the following:
-                  Missile
-                  Scan Visor and Combat/Scan Dash (Intermediate)
-          All of the following:
-              Space Jump Boots
-              Missile or Jump Off Enemies (Beginner)
+      All of the following:
+          L-Jump (Beginner)
```

Comment:
This forces a trick to climb the room, which isn't correct.

Reply:
Moved the expert jump of enemies to the "any of the following" section